modules/ROOT/pages/troubleshoot-api-provisioning-issues.adoc

```diff
@@ -0,0 +1,54 @@
+= Troubleshooting API Provisioning Issues
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, API provisioning issues, troubleshooting
+
+After you provision an adapter, you must create an API, to which you then bind the service. You might encounter errors when you provision an API if you provide invalid credentials, and so on. 
```

Comment:
```suggestion
After you provision an adapter, you must create an API, to which you then bind the service. You encounter errors when you provision an API if you provide invalid credentials. 
```

Reply:
This is not a "might", right? You WILL encounter errors with invalid credentials?
This seems like kind of a nontopic. We are documenting troubleshooting a pretty simple user error: garbage in, garbage out!
Do you suppose that these troubleshooting topics could be recast as something like...:
If you encounter issues with API provisioning, ensure that:
* Your credentials are valid
* Some other reason
In this case, you would not need a list but just a simple sentence. Some troubleshooting does not really require separate Cause, Diagnose, and Solution categories (in fact, I prefer Issue (what the user sees or experiences), Cause, and Solution (or Recovery).

Reply:
Will ask!